src/lib/constant_mappings.cpp

```diff
@@ -35,6 +36,19 @@ const boost::bimap<PredicateCondition, std::string> predicate_condition_to_strin
         {PredicateCondition::IsNotNull, "IS NOT NULL"},
     });
 
+const std::unordered_map<PredicateCondition, ExpressionType> predicate_condition_to_expression_type = {
```

Comment:
Brrr. This is not something that comes from JIT, but brrr. @mrzzzrm, can you think of any way to avoid this? Unfortunately, we cannot extend enums. :/

Reply:
if a enum has an intersection with another, some people make them use the same values in their intersection

````enum class Foo {A,B,C}; enum class Bar { A = Foo::A, B = Foo::B, C}; ```

but I'd rather see this as yet another reason to get rid of our Expression-mess